saleor/core/utils/validators.py

```diff
@@ -41,6 +41,6 @@ def user_is_valid(user: Optional[User]) -> bool:
     return bool(user and not user.is_anonymous)
 
 
-def date_passed(given_date):
-    """Return true when date has passed."""
-    return given_date < date.today()
+def date_in_future(given_date):
```

Comment:
`is_date_in_future` would be a better name- it suggests that it runs some checks and returns a boolean.